### PR TITLE
Verify number of lines

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
Good idea but can you turn this into an actual test file that can also be run from shell? If not pytest (even though eventually we need to setup pytest here too) at least a shell script.

File: tests/count_last_changes.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# ====
+# Check that the last changes increment correctly the number of updates to the last release
+# in the info.json file
+# ====
+# Get the last changes folder
+latest=$(ls updates| sort -n | tail -n1)
+# Count the number of lines in the updates files
+lines=$(sudo wc -l updates/$latest/updates.sql | cut -d' ' -f1)
```
Comment (LefterisJP) [inline tests/count_last_changes.sh:9]: you should definitely not need `sudo` for a wc

**Review by LefterisJP — COMMENTED** [review]
you need to make it executable also `chmod +x tests/count_last_changes.sh`

Later you can also add a tool under a new `tools` directory (probably a python script) to auto fill the json file too.

File: tests/count_last_changes.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# Obtain from the info.json file the number of updates for the last version
+expected=$(cat updates/info.json | jq --arg latest "$latest" '.updates[$latest]')
+if [ $expected -eq $(expr $lines / 2) ];then
+    exit 0
+else
+    echo "Expected" $expected "changes but got" $(expr $lines / 2)
```
Comment (LefterisJP) [inline tests/count_last_changes.sh:15]: Give more information here. Mention the name of the file "info.json"

File: tests/count_last_changes.sh (hunk 1 above)
Comment (yabirgb) [reply tests/count_last_changes.sh:9]: I don't know why before it was failing without sudo but it's true that is not needed. Updated

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**